doc/book/in-depth-guide/zend-db-sql-zend-hydrator.md

```diff
@@ -398,7 +398,7 @@ use Zend\Db\ResultSet\ResultSet;
 Next, we'll add the following import statements to our class file:
 
 ```php
-use Zend\Hydrator\Reflection as ReflectionHydrator;
+use Zend\Hydrator\ReflectionHydrator;
```

Comment:
The problem with this change is that is still (and only one) valid for zend-hydrator v2.